apps/coscrad-frontend/src/app/App.spec.tsx

```diff
@@ -1,12 +1,19 @@
 import { render } from '@testing-library/react';
+import { Provider } from 'react-redux';
 import { MemoryRouter } from 'react-router-dom';
+import { store } from '../store';
 import App from './App';
 
-
 describe('App', () => {
-  it('should render successfully', () => {
-    const { baseElement } = render(<MemoryRouter><App /></MemoryRouter>);
+    it('should render successfully', () => {
+        const { baseElement } = render(
+            <MemoryRouter>
+                <Provider store={store}>
```

Comment:
We probably don't want the real store here. We don't want the redux thunks firing off async in Jest tests. Save that for Cypress.